homepage/templates/homepage/app_layout.html

```diff
@@ -32,8 +37,8 @@
         <a class="nav-link" href="/reviews">Reviews</a>
       </li>   
       <li class="nav-item">
-        <a class="nav-link disabled" href="#">Sign In</a>
-      </li>
+        <a class="nav-link" href="/users/sign_in">Sign In</a>
+      </li>        

```

Comment:
looks like you have extra spaces in the end of the line please remove them
```suggestion
      </li> 

```

Reply:
removed